proxies/live/apiproxy/policies/KeyValueMapOperations.GetSecureVariables.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<KeyValueMapOperations async="false" continueOnError="false" enabled="true" name="KeyValueMapOperations.GetSecureVariables" mapIdentifier="pds-variables-encrypted">
```

Comment:
Is this policy supposed to be here, and should we really be using PDS variables?